joomla/joomla-cms
# Add profile link to login module

The login module has links to creating an account and password/username reminders
The logout only has a greeting and a button

### Summary of Changes
This PR add a link to the profile page in the logout view. Currently the only way to view your profile and optionaly edit that profile is if there is a dedicated menu link. This removes the need for that menu item and just adds it to the module

![sn7u 1](https://cloud.githubusercontent.com/assets/1296369/19658491/117df3d0-9a20-11e6-84e9-a3e49e01b53d.png)

### Testing Instructions
Apply the patch and login on the front end of your site and you will see the new option

### Backwards Compatibility
Is adding a new link to the front end a break in B/C ? I dont know. If it is felt that it is then we could always add a parameter to the module to enable this link (it would therefore have to be disable by default) but I'd rather not add another param unless its felt its really required.
